pywbem/mof_compiler.py

```diff
@@ -563,9 +665,7 @@ def p_mp_createClass(p):
                             if embedded_inst.value not in dep_classes:
                                 dep_classes[embedded_inst.value] = ce
 
-                            continue
-
-                    for cln, err in dep_classes.items():
+                    for cln, _ in dep_classes.items():
```

Comment:
Am I missing something.  If the value rtn is _ why are we processing as dep_classes.items() rather than just dep_classes to retrieve each cln???

Reply:
Because I was sloppy... If you agree not to show the original exception text, we can even change it to a list.

Reply:
DONE - changed dict to a list.